dotCMS/src/main/java/com/dotcms/saml/DotIdentityProviderConfigurationImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.dotcms.saml;
+
+import com.dotcms.saml.service.external.IdentityProviderConfiguration;
+import com.dotcms.saml.service.external.SamlName;
+import com.dotcms.security.apps.AppSecrets;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Map;
+
+/**
+ * Default implementation to retrieve the configuration from apps
+ * @author jsanca
+ */
+public class DotIdentityProviderConfigurationImpl implements IdentityProviderConfiguration {
+
+    private final AppSecrets appSecrets;
+
+    public DotIdentityProviderConfigurationImpl(final AppSecrets appSecrets) {
+
+        this.appSecrets = appSecrets;
+    }
+
+    @Override
+    public boolean isEnabled() {
+
+        final String enableKey = SamlName.DOT_SAML_ENABLE.getPropertyName();
+        return this.appSecrets.getSecrets().containsKey(enableKey)?
```

Comment:
Codacy found an issue: [Ternary operators that can be simplified with || or &&](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)